libs/features/personalization/personalization.js

```diff
@@ -1059,7 +1081,7 @@ export async function init(enablements = {}) {
       loadLink(normalizedURL, { as: 'fetch', crossorigin: 'anonymous', rel: 'preload' });
     });
   }
-
+  preloadMarqueeImage();
```

Comment:
@denlight, if the intention is to only use this new feature when waiting for the Target response, you could move this call to line 1085 "target === true" condition to only fire when target is on.